doc/api/deprecations.md

```diff
@@ -3056,6 +3056,24 @@ This method was deprecated because it is not compatible with
 
 Use [`buffer.subarray`][] which does the same thing instead.
 
+### DEP0159: `process._getActiveRequests()` and `process._getActiveHandles()`
```

Comment:
```suggestion
### DEP0XXX: `process._getActiveRequests()` and `process._getActiveHandles()`
```

The deprecation number is assigned when the PR lands

Reply:
Except that it's no longer the case since the CQ uses "Squash and merge" instead of pushing a different commit.